### PR TITLE
COTECH-975 Correct false positive test results in PlayerSetup

### PR DESCRIPTION
**Pull request by andrzejkupczyk**
[COTECH-975](https://fandom.atlassian.net/browse/COTECH-975) @Wikia/cotech 

[COTECH-975]: https://fandom.atlassian.net/browse/COTECH-975?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: spec/platforms/shared/setup/player.setup.spec.ts [hunk 1]
```diff
@@ -54,71 +57,68 @@ describe('PlayerSetup', () => {
...
 		context.set('slots.featured.videoAdUnit', MOCKED_VAST_AD_UNIT);
 		context.set('vast.adUnitId', MOCKED_VAST_AD_UNIT);
-
+		const slotName = 'featured';
 		const expectedDispatchArg = {
 			showAds: true,
 			autoplayDisabled: false,
 			videoAdUnitPath: MOCKED_VAST_AD_UNIT,
-			targetingParams: 'src=test',
+			targetingParams: 'experiment_groups=strategy_rules_undefined&pos=featured&rv=1',
```
Comment (andrzejkupczyk) [inline spec/platforms/shared/setup/player.setup.spec.ts:112]: I wasn't entirely sure about this change. Does it look okay to you, or did I overlook something?

**Review by dzawada — APPROVED** [review]

File: spec/platforms/shared/setup/player.setup.spec.ts (hunk 1 above)
Comment (dzawada) [reply spec/platforms/shared/setup/player.setup.spec.ts:112]: If the test is passing I would say it's fine :D

Comment (andrzejkupczyk) [reply spec/platforms/shared/setup/player.setup.spec.ts:112]: Funny thing – they were actually passing before I made my changes too 😄

Comment (nandy-andy) [reply spec/platforms/shared/setup/player.setup.spec.ts:112]: Looks good 👍

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**